nixos/modules/security/please.nix

```diff
@@ -0,0 +1,105 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.security.please;
+  ini = pkgs.formats.ini { };
+in
+{
+  options.security.please = {
+    enable = mkEnableOption (mdDoc ''
+      please, a tool which allows non-root users to execute commands as root
```

Comment:
Just as `root`? If so, not really a `sudo` clone.

Reply:
Clarified that part.